### ISSUE TITLE
Allow skipping to timestamp when clicking on lyrics line

### ISSUE THREAD

**Issue by angeloevangelista** [angeloevangelista/lyrics-for-spotify#3]

**Comment by oskarbukovsky** [comment]
I AGREE!!, if app knows when to move to next line, you can get timestamp of the line user clicks and than modify playback time in the song. But i bet it can be tricky with spotify with what they are doing last months, on free tier on android its almost unusable nevertheless. 
Also great project ❤️
